### PR TITLE
update volume docs(Ja)

### LINKED ISSUE #21307: ja: Make glossary/volume.md follow v1.17 of the original text
**This is a Feature Request**

**What would you like to be added**
Update glossary/volume.md to follow v1.17 of the original (Englist) text.

**Why is this needed**
content/ja/docs/reference/glossary/volume.md it outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/blob/dev-1.17-ja.2/content/ja/docs/reference/glossary/volume.md

Original
https://github.com/kubernetes/website/blob/release-1.17/content/en/docs/reference/glossary/volume.md

diff between translated and v1.17
https://gist.github.com/oke-py/941bfb5156d022507f4fc61e687ec9d1

**Note**
Currently, we use `dev-1.17-ja.2` branch. Please open a PR targeting the branch.

ref
[How To Contribute](https://kubernetes-docs-ja.kibe.la/shared/entries/c5878aa5-ad1f-4f29-a5bb-25853cbc14ec)
[翻訳スタイルガイド](https://kubernetes-docs-ja.kibe.la/shared/entries/5efe4fa7-d2a1-4a1d-8bc3-ce7ccdc064a6)

If you have a question, feel free to ask us here or slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by ytakaya**
fixes #21307

### CODE REVIEW

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
I signed it

**Comment by inductor** [comment]
@ytakaya please sign the CLA :)

**Comment by ytakaya** [comment]
I signed it

**Comment by inductor** [comment]
@ytakaya maybe your linux foundation account and github account do not match? 👀

**Comment by ytakaya** [comment]
@inductor I signed up for linux foundation with this github account...

**Comment by inductor** [comment]
@ytakaya what about email?

**Comment by oke-py** [comment]
@ytakaya Note that you need to sign 2 types of CLA. Have you done them?

**Comment by ytakaya** [comment]
I signed it

**Comment by ytakaya** [comment]
@inductor @oke-py 
Oh, I see.
I had only done individual signup, and now I have done Corporation signup.

**Comment by oke-py** [comment]
@ytakaya Uh, probably only individual signup is enough.
Have you received a completion e-mail from Linux Foundation?

The 2nd CLA is CNCF's CLA.
Have you signed a form like this?
https://github.com/cncf/cla/blob/master/corporate-cla.pdf

**Comment by ytakaya** [comment]
@oke-py Yes, I signed it and received a e-mail with "You just signed CNCF Corporate Contributor License Agreement" in the subject line.

**Comment by oke-py** [comment]
Corporate?
The account who signed CLA matches the account related to the commit?

**Comment by ytakaya** [comment]
@oke-py Yes, I signed by this account.
The two types of CLA mean the following??

- Individual signup to sign up as an individual or as an employee of a signed organization.
- Corporation signup to sign as a corporation representative and manage signups from your organization.

from this page.
https://git.k8s.io/community/CLA.md#the-contributor-license-agreement

**Comment by oke-py** [comment]
@ytakaya 2 types means Linux Foundation CLA and CNCF CLA.
Actually, long time passed since I signed CLA, and I can't remember exactly.

**Comment by ytakaya** [comment]
@oke-py Maybe it's done now. Could you please confirm?

**Comment by oke-py** [comment]
Good! I'll start review.
Now I found CNCF CLA's URL includes corporate. 2 types might mean individual and corporate. Umm

**Comment by ytakaya** [comment]
@inductor @oke-py Thank you so much for your help!!

File: content/ja/docs/reference/glossary/volume.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ tags:
 - core-object
 - fundamental
 ---
- {{< glossary_tooltip text="ポッド" term_id="pod" >}}内のコンテナからアクセス可能なデータを含むディレクトリ。
+ {{< glossary_tooltip text="ポッド" term_id="pod" >}}内の{{< glossary_tooltip text="containers" term_id="container" >}}からアクセス可能なデータを含むディレクトリ。
```
Comment (oke-py) [inline content/ja/docs/reference/glossary/volume.md:14]:
```suggestion
 {{< glossary_tooltip text="Pod" term_id="pod" >}}内の{{< glossary_tooltip text="コンテナ" term_id="container" >}}からアクセス可能なデータを含むディレクトリ。
```
glossary_tooltipのtextも訳しますが、ポッドはPod表記に統一します。

File: content/ja/docs/reference/glossary/volume.md [hunk 2]
```diff
@@ -11,8 +11,10 @@ tags:
...
 
 <!--more--> 
 
-Kubernetesボリュームはボリュームを含む{{< glossary_tooltip text="ポッド" term_id="pod" >}}が存在する限り有効です。そのためボリュームは{{< glossary_tooltip text="ポッド" term_id="pod" >}}内で実行されるすべての{{< glossary_tooltip text="コンテナ" term_id="container" >}}よりも長持ちし、{{< glossary_tooltip text="コンテナ" term_id="container" >}}の再起動後もデータは保持されます。
+Kubernetesボリュームはボリュームを含むポッドが存在する限り有効です。そのためボリュームはポッド内で実行されるすべてのコンテナよりも長持ちし、コンテナの再起動後もデータは保持されます。
```
Comment (oke-py) [inline content/ja/docs/reference/glossary/volume.md:18]:
```suggestion
Kubernetesボリュームはボリュームを含むPodが存在する限り有効です。そのためボリュームはPod内で実行されるすべてのコンテナよりも長持ちし、コンテナの再起動後もデータは保持されます。
```

**Comment by oke-py** [comment]
Files Changedに出てこない箇所にも1つ"ポッド"がありますね。そちらも"Pod"に修正お願いいたします。

**Comment by ytakaya** [comment]
@oke-py 修正しました．確認よろしくお願いいたします．

File: content/ja/docs/reference/glossary/volume.md [hunk 3]
```diff
@@ -4,15 +4,17 @@ id: volume
...
-  ポッド内のコンテナからアクセス可能なデータを含むディレクトリ。
+  Pod内のコンテナからアクセス可能なデータを含むディレクトリ。
 
 aka: 
 tags:
 - core-object
 - fundamental
 ---
- {{< glossary_tooltip text="ポッド" term_id="pod" >}}内のコンテナからアクセス可能なデータを含むディレクトリ。
+ {{< glossary_tooltip text="Pod" term_id="pod" >}}内の{{< glossary_tooltip text="containers" term_id="container" >}}からアクセス可能なデータを含むディレクトリ。
```
Comment (oke-py) [inline content/ja/docs/reference/glossary/volume.md:14]:
```suggestion
 {{< glossary_tooltip text="Pod" term_id="pod" >}}内の{{< glossary_tooltip text="コンテナ" term_id="container" >}}からアクセス可能なデータを含むディレクトリ。
```

**Comment by inductor** [comment]
@ytakaya Thank you for your contribution!

**Pull request merged**